src/js/components/Table/StyledTable.js

```diff
@@ -67,7 +67,14 @@ const StyledTableCell = styled.td.withConfig(styledComponentsConfig)`
 const StyledTableDataCaption = styled.caption.withConfig(
   styledComponentsConfig,
 )`
-  margin-bottom: ${(props) => props.theme.global.edgeSize.xxsmall};
+  ${(props) =>
+    edgeStyle(
+      'margin',
+      props.theme.table.caption.margin,
+      props.responsive,
```

Comment:
I wonder if we should include `props.responsive` since we don't document this as an available option